src/admin-views/attendees/attendees-event/overview.php

```diff
@@ -56,7 +56,7 @@ class="tec-tickets__admin-attendees-overview-ticket-type-icon <?php echo esc_att
 							<?php esc_html_e( $ticket->name ); ?>
 						</span>
 						<span class="tec-tickets__admin-attendees-overview-ticket-type-list-item-ticket-id">
-							<?php esc_html_e( sprintf( '#%d', $ticket->ID ) ); ?>
+							<?php echo esc_html( sprintf( '#%d', $ticket->ID ) ); ?>
```

Comment:
```suggestion
							<?php echo (int) $ticket->ID; ?>
```

@DAnn2012 this seems like it could be simplified even more, since the ID property must be an integer.

Reply:
Thanks!